integration/run_integration_test.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+#
+# run_integration_test.sh is a small wrapper around the run_test function.
+# It's intended to be used by the Travis CT/Trillian integration test only,
+# and will go away when we migrate off of Travis.
+set -e
+
+INTEGRATION_DIR="$( cd "$( dirname "$0" )" && pwd )"
+. "${INTEGRATION_DIR}"/functions.sh
+
+run_test "$1" "$2"
+
+# See if this helps debug the deadlocks we keep seeing.
```

Comment:
This didn't provide any useful info and adds log spam. Can remove it now if you want.

Reply:
ah, cool - ta!